extensions/s3-extensions/src/main/java/io/druid/storage/s3/S3StorageDruidModule.java

```diff
@@ -39,6 +39,7 @@
 public class S3StorageDruidModule implements DruidModule
 {
   public static final String SCHEME = "s3_zip";
+  public static final String URI_SCHEME = "s3";
```

Comment:
do we need similar changes in all the Storage modules ? I see a separate PR for local storage, what about others, HDFS, cassandra ?